src/commands/lint.ts

```diff
@@ -24,6 +24,14 @@ export class LintCommand implements Command {
 
   args = [
     {
+      name: 'input',
```

Comment:
It seems to be a common usecase for people getting started, or debugging linting issues. Being able to do `polymer lint path/to/file.html` is mentioned in both https://github.com/Polymer/polymer-cli#lint & https://www.polymer-project.org/1.0/docs/tools/polymer-cli#lint.

@robdodson might be able to shed some user-land light on this